libdnf/solv/pool.hpp

```diff
@@ -232,17 +234,16 @@ class Pool {
 
     const SolvMap & get_considered_map() const noexcept { return considered; }
 
-    void set_considered_map(SolvMap && considered_map) {
+    void swap_considered_map(SolvMap & considered_map) {
```

Comment:
I guess it will be nice to have a description what it does what is expected from the argument.

Reply:
Ok. Description added.